org.adempiere.base/src/org/compiere/model/GridField.java

```diff
@@ -444,8 +444,30 @@ public boolean isEditable (Properties ctx, boolean checkContext,boolean isGrid)
 		}
 
 		//  Fields always updateable
-		if (m_vo.IsAlwaysUpdateable)      //  Zoom
-			return true;
+		if (m_vo.IsAlwaysUpdateable)

```

Comment:
Bringing comment from @hengsin in ticket:

> AlwaysUpdatableLogic is only check and override IsAlwaysUpdateable when IsAlwaysUpdateable is true. This is inconsistent with how current code deals with IsReadonly and ReadonlyLogic. I don’t think such inconsistency is a good thing to have

Reply:
This also resolved by making isAlwaysUpdatable flag has more preference than updatable logic